.kokoro/test.sh

```diff
@@ -13,27 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-EXIT_STATUS=0
+# Fail only any non-zero status code
+set -e
 
 pushd $(dirname "$0")/../
 
-for directory in `find clients -mindepth 3 -maxdepth 3 -type d | sort`
+VARIANT="2.0.0"
+
+# Only search for directories with the latest variant as only the generator only
+# generates libraries for the latest variant
+for directory in `find clients -mindepth 3 -maxdepth 3 -type d | grep ${VARIANT} | sort`
 do
   pushd $directory
-  diff=$(git diff master .)
+  diff=$(git diff main .)
```

Comment:
```
error: Could not access 'main'
```

Reply:
Modified to `git diff "${KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH}...${KOKORO_GITHUB_PULL_REQUEST_COMMIT}" -- "${directory}"`

Should only find diffs that are in this PR branch and in the directory